docs/docs/concepts/project_codes.md

```
@@ -12,9 +12,11 @@ Individual orders (such as [Purchase Orders](../purchasing/purchase_order.md) or
 
 Management of project codes (such as creating or editing codes) is accessed via the [settings page](../settings/global.md). Select the *Project Codes* tab to access project code configuration options:
 
-{% with id="project_codes", url="settings/project_codes.png", description="Project Codes List" %}
-{% include "img.html" %}
-{% endwith %}
+{{ image("settings/project_codes.png", "Project Codes") }}
+
+| Name | Description | Default | Units |
+| ---- | ----------- | ------- | ----- |
+{{ globalsetting("PROJECT_CODES_ENABLED") }}
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)